### DISCUSSION TITLE
CfP FOSDEM Software Defined Storage devrrom

### DISCUSSION THREAD

**Discussion by jan--f** [Seagate/cortx#505] [General] [answered]
FOSDEM is a free software event that offers open source communities a place to
meet, share ideas and collaborate.  It is well known for being highly
developer-oriented and in the past brought together 8000+ participants from
all over the world.  It's home is in the city of Brussels (Belgium).

FOSDEM 2021 will take place as an online event during the weekend of February
6./7. 2021. More details about the event can be found at http://fosdem.org/

** Call For Participation

The Software Defined Storage devroom will go into it's fifth round for talks
around Open Source Software Defined Storage projects, management tools
and real world deployments.

Presentation topics could include but are not limited too:

- Your work on a SDS project like Ceph, Gluster, OpenEBS, CORTX or Longhorn

- Your work on or with SDS related projects like OpenStack SWIFT or Container
Storage Interface

- Management tools for SDS deployments

- Monitoring tools for SDS clusters

** Important dates:

- Dec 27th 2020:  submission deadline for talk proposals
- Dec 31st 2020:  announcement of the final schedule
- Feb  6th 2021:  Software Defined Storage dev room

Talk proposals will be reviewed by a steering committee:
- Niels de Vos (OpenShift Container Storage Developer - Red Hat)
- Jan Fajerski (Ceph Developer - SUSE)
- TBD

Use the FOSDEM 'pentabarf' tool to submit your proposal:
https://penta.fosdem.org/submission/FOSDEM21

- If necessary, create a Pentabarf account and activate it.
Please reuse your account from previous years if you have
already created it.
https://penta.fosdem.org/user/new_account/FOSDEM21

- In the "Person" section, provide First name, Last name
(in the "General" tab), Email (in the "Contact" tab)
and Bio ("Abstract" field in the "Description" tab).

- Submit a proposal by clicking on "Create event".

- If you plan to register your proposal in several tracks to increase your
chances, don't! Register your talk once, in the most accurate track.

- Presentations have to be pre-recorded before the event and will be streamed
on   the event weekend.

- Important! Select the "Software Defined Storage devroom" track
(on the "General" tab).

- Provide the title of your talk ("Event title" in the "General" tab).

- Provide a description of the subject of the talk and the
intended audience (in the "Abstract" field of the "Description" tab)

- Provide a rough outline of the talk or goals of the session (a short
list of bullet points covering topics that will be discussed) in the
"Full description" field in the "Description" tab

- Provide an expected length of your talk in the "Duration" field.
  We suggest a length between 15 and 45 minutes.

** For accepted talks

Once your proposal is accepted we will assign you a volunteer deputy who will
help you to produce the talk recording.  The volunteer will also try to ensure
the recording is of good quality, help with uploading it to the system,
broadcasting it during the event and moderate the Q&A session after the
broadcast.  Please note that as a presenter you're expected to be available
online during and especially after the broadcast of you talk.  The schedule
will be available under
https://fosdem.org/2021/schedule/track/software_defined_storage/

Hope to hear from you soon! And please forward this announcement.

If you have any further questions, please write to the mailing list at
storage-devroom@lists.fosdem.org and we will try to answer as soon as
possible.

Thanks!

**Comment by johnbent** [comment, marked as answer]
Thanks @jan--f !  We will definitely submit something and will share this with our contacts.  Looking forward to "meeting" you at FOSDEM!
